packages/universal-components/src/RangeDatePicker/DraggableItem.ios.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+
+import React from 'react';
+import { Animated } from 'react-native';
+import { PanGestureHandler, State } from 'react-native-gesture-handler';
```

Comment:
This should be listed in dependencies. It is preinstalled in expo, but will cause problem when someone try to use universal components without expo.

Reply:
OK, done, it's already in Expo, so I added peerDependency to `universal-components`.

Reply:
Ok, thanks. But do we need to add whole `expo` as dependency? Wouldn't be just previous `react-native-gesture-handler` enough?

Reply:
Right, if we use expo `react-native-gesture-handler` it shouldn't be installed again, but it's pity to add whole Expo dependency to universal-components. So I added `react-native-gesture-handler` to peerDependency and it should be fine now.